plugin.xml

```diff
@@ -56,8 +53,7 @@
             <source url="https://cdn.cocoapods.org/"/>
         </config>
         <pods use-frameworks="true">
-            <pod name="FirebaseMessaging" spec="10.23.0" />
-            <pod name="OSCommonPluginLib" spec="1.0.0" />
+            <pod name="FirebaseMessaging" spec="10.29.0" />
```

Comment:
Should this update also be mentioned in the `CHANGELOG.md` file?

Reply:
Added on both lib and bridge.